tilmeld.html

```diff
@@ -28,7 +28,56 @@
     </header>
 
     <main>
+        <section class="banner">
+            <div class="banner__content">
+                <img class="banner__photo" src="assets/images/mobile/tilmeld-banner-mobile.webp"
+                    alt="mere end dyrevenner banner">
+                <div class="overlay__text">
+                    <p class="tilmeld__h1">TILMELD</p>
+                </div>
+            </div>
+        </section>
+        <section class="section-white">
+            <div class="section-white__content">
+                <h3 class="section-white__h3">Sådan foregår kampagnen</h3>
+                <p>Dyrenes Beskyttelse inviterer dig til en hyggedag på dit lokale internat! Her kan du møde både dyrene
+                    og andre unge, der måske har det ligesom dig.
+                    <br>
+                    <br>
+                    Dagen handler om at skabe nye forbindelser – både med pelsede venner og nye bekendtskaber.
+                    <br>
+                    <br>
+                    Ensomhed kan ramme os alle, og det kan være lettere at åbne op med søde kæledyr som icebreakere.
+                    <br>
+                    <br>
+                    Det er en uformel og tryg ramme, hvor alle er velkomne!
+                </p>
+
+            </div>
+
+            <div class="dropdown-boxes">
+                <div class="dropdown-forventning">
+                    <p class="dropdown-forventning__text">Hvad kan du forvente ved at deltage?</p>
+                    <img src="assets/illustrations/mobile/arrow-down-large-blue-mobile.svg" alt="arrow-down">
+                </div>
+                <div class="dropdown-praktisk">
+                    <p class="dropdown-praktisk__text">Hvad kan du forvente ved at deltage?</p>
+                    <img src="assets/illustrations/mobile/arrow-down-large-blue-mobile.svg" alt="arrow-up">
+                </div>
```

Comment:
Fordi at "forventning" og "praktisk" er ens elementer så kan vi bruge den samme class på den. Navnet kunne bare være "dropdown-box". Herunder kan du se hvad jeg mener. Husk også at ændre det i CSS filen hvis det ændres til det. 

```suggestion
                <div class="dropdown-box">
                    <p class="dropdown-box__text">Hvad kan du forvente ved at deltage?</p>
                    <img src="assets/illustrations/mobile/arrow-down-large-blue-mobile.svg" alt="arrow-down">
                </div>
                <div class="dropdown-box">
                    <p class="dropdown-box__text">Hvad kan du forvente ved at deltage?</p>
                    <img src="assets/illustrations/mobile/arrow-down-large-blue-mobile.svg" alt="arrow-up">
                </div>
```